libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -174,6 +177,24 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
     return this.#_checked;
   }
 
+  /**
+   * Indicates whether the checkbox is in the indeterminate state. This has no visual effect for icon checkboxes.
+   * @internal
+   */
+  @Input()
+  public set indeterminate(value: boolean | undefined) {
+    this.#_indeterminate = !!value;
+    this.#indeterminateChange.next(this.#_indeterminate);
+    if (this.inputEl) {
+      this.inputEl.nativeElement.indeterminate = this.#_indeterminate;
```

Comment:
The native `indeterminate` state can currently only be set via TypeScript/JavaScript which is why I am setting this in the `setter` instead of in the HTML.